elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -5,6 +5,7 @@ $LOCAL: cta;
  
 $pfe-cta--BackgroundColor--focus: rgba(40, 151, 240, 0.2);
 $pfe-cta--Color--fallback: #003366;
+$pfe-cta--FontSize--default: calc(#{pfe-local($cssvar: FontSize)} * 1.125);
```

Comment:
Is this still being used? I don't see it in the diff.

Reply:
Thanks for looking @castastrophe 
you're correct. I removed this sass var.